roda-common/roda-common-data/src/main/java/org/roda/core/data/common/RodaConstants.java

```diff
@@ -159,8 +159,10 @@ public enum DateGranularity {
   public static final String UI_MARKETPLACE_URL = "ui.url.marketplace";
   public static final String UI_MONITORING_URL = "ui.url.monitoring";
   public static final String UI_REPORTING_URL = "ui.url.reporting";
-  public static final String UI_MONITORING_ACTIVE = "ui.menu.administration.reporting";
-  public static final String UI_REPORTING_ACTIVE = "ui.menu.administration.reporting";
+  public static final String UI_MONITORING_STATUS = "ui.service.reporting";
+  public static final String UI_REPORTING_STATUS = "ui.service.reporting";
```

Comment:
These properties should be different from each other.

Reply:
done